src/tests/opaque-munit.c

```diff
@@ -324,7 +324,7 @@ MunitResult opaque_test(const MunitParameter params[], void* user_data_or_fixtur
     return MUNIT_FAIL;
   }
 
-  printf("all ok\n");
+  printf("\n");
```

Comment:
why this? `all ok` is a nice visual feedback at the end of a test. otherwise the return value must be checked, no?

Reply:
For opaque-test.c, "all ok" is needed.

For opaque-munit.c, "all ok" is not needed. Having **"`[ OK    ]`"** or **"`[ FAIL  ]`"** in the output makes it clear if the test succeeded or failed.

Before removing "all ok", here is the output:
```
# SUCCESS
LD_LIBRARY_PATH=. ./tests/opaque-munit --fatal-failures
Running test suite with seed 0x90544508...
/opaque-tests/server-init            
  pw=simple guessable dictionary password, key=some optional key contributed to the opaque protocol, idU=user, idS=server, cfg=, type=all ok
[ OK    ] [ 0.44007080 / 0.42313828 CPU ]
  pw=simple guessable dictionary password, key=some optional key contributed to the opaque protocol, idU=user, idS=server, cfg=, type=all ok
[ OK    ] [ 0.48506300 / 0.44730324 CPU ]
  pw=simple guessable dictionary password, key=some optional key contributed to the opaque protocol, idU=user, idS=server, cfg=, type=all ok
[ OK    ] [ 0.38269270 / 0.35494423 CPU ]
  pw=simple guessable dictionary password, key=some optional key contributed to the opaque protocol, idU=user, idS=server, cfg=, type=all ok
[ OK    ] [ 0.37857840 / 0.36205318 CPU ]
  pw=simple guessable dictionary password, key=some optional key contributed to the opaque protocol, idU=user, idS=server, cfg=, type=all ok
[ OK    ] [ 0.36918350 / 0.35275971 CPU ]
...

# FAILURE
LD_LIBRARY_PATH=. ./tests/opaque-munit --fatal-failures
Running test suite with seed 0xf120ac90...
/opaque-tests/server-init            
  pw=simple guessable dictionary password, key=some optional key contributed to the opaque protocol, idU=user, idS=server, cfg=, type=[ FAIL  ]
cfg sku: 0, pku:0, pks:1, idu:0, ids:0
sizeof(rec): 235

opaque_CreateRegistrationRequest

opaque_CreateRegistrationResponse

opaque_FinalizeRequest

opaque_StoreUserRecord

opaque_CreateCredentialRequest

opaque_CreateCredentialResponse

opaque_RecoverCredentials
failed authenticating user
```

After removing "all ok", here is the output:
```
# SUCCESS
LD_LIBRARY_PATH=. ./tests/opaque-munit --fatal-failures
Running test suite with seed 0x5cda29f8...
/opaque-tests/server-init            
  pw=simple guessable dictionary password, key=some optional key contributed to the opaque protocol, idU=user, idS=server, cfg=, type=
[ OK    ] [ 0.39631970 / 0.37637588 CPU ]
  pw=simple guessable dictionary password, key=some optional key contributed to the opaque protocol, idU=user, idS=server, cfg=, type=
[ OK    ] [ 0.34991170 / 0.32362322 CPU ]
  pw=simple guessable dictionary password, key=some optional key contributed to the opaque protocol, idU=user, idS=server, cfg=, type=
[ OK    ] [ 0.36073790 / 0.34333849 CPU ]
  pw=simple guessable dictionary password, key=some optional key contributed to the opaque protocol, idU=user, idS=server, cfg=, type=
[ OK    ] [ 0.34249840 / 0.31755360 CPU ]
  pw=simple guessable dictionary password, key=some optional key contributed to the opaque protocol, idU=user, idS=server, cfg=, type=
[ OK    ] [ 0.32464460 / 0.30986667 CPU ]
...

# FAILURE
LD_LIBRARY_PATH=. ./tests/opaque-munit --fatal-failures
Running test suite with seed 0xb3ce820e...
/opaque-tests/server-init            
  pw=simple guessable dictionary password, key=some optional key contributed to the opaque protocol, idU=user, idS=server, cfg=, type=[ FAIL  ]
cfg sku: 0, pku:0, pks:1, idu:0, ids:0
sizeof(rec): 235

opaque_CreateRegistrationRequest

opaque_CreateRegistrationResponse

opaque_FinalizeRequest

opaque_StoreUserRecord

opaque_CreateCredentialRequest

opaque_CreateCredentialResponse

opaque_RecoverCredentials
failed authenticating user
0 of 1 (0%) tests successful, 0 (0%) test skipped.
```

Reply:
ah yes makes total sense